packages/wallets/src/features/accounts/screens/PersonalDetails/PersonalDetails.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { ReactNode } from 'react';
+import { useResidenceList } from '@deriv/api';
+import { WalletDropdown, WalletText, WalletTextField } from '../../../../components';
+import { accountOpeningReasonList } from './constants';
+import './PersonalDetails.scss';
+
+const PersonalDetails = () => {
+    const { data: residenceList } = useResidenceList();
+
+    return (
+        <div className='wallets-personal-details'>
+            <div className='wallets-personal-details__header'>
+                <WalletText align='center' as='h2' color='prominent' weight='bold'>
+                    Complete your personal details
+                </WalletText>
+                <WalletText align='center' size='xs'>
+                    Any information you provide is confidential and will be used for verification purposes only.
+                </WalletText>
+            </div>
+            <div className='wallets-personal-details__form'>
+                <WalletDropdown
+                    label='Citizenship*'
+                    list={residenceList.map(residence => ({
+                        text: residence.text as ReactNode,
+                        value: residence.value ?? '',
+                    }))}
+                    maxWidth={'40rem'}
+                    onSelect={() => null}
+                    type='comboBox'
+                    value={undefined}
+                />
+                <WalletDropdown
+                    label='Place of birth*'
+                    list={residenceList.map(residence => ({
+                        text: residence.text as ReactNode,
+                        value: residence.value ?? '',
+                    }))}
+                    maxWidth='40rem'
+                    onSelect={() => null}
+                    type='comboBox'
+                    value={undefined}
+                />
+                <WalletDropdown
+                    label='Tax residence*'
+                    list={residenceList.map(residence => ({
+                        text: residence.text as ReactNode,
+                        value: residence.value ?? '',
+                    }))}
+                    maxWidth='40rem'
+                    onSelect={() => null}
+                    type='comboBox'
+                    value={undefined}
+                />
+                <WalletTextField
+                    label='Tax identification number*'
+                    maxWidth='40rem'
+                    onChange={() => null}
+                    value={undefined}
+                />
+                <WalletDropdown
+                    label='Account opening reason*'
+                    list={accountOpeningReasonList}
+                    maxWidth='40rem'
```

Comment:
as per other discussions - maxWidth is a css concern, not javascript concern, otherwise we can start as well just using inline styles all over and completely ditch the idea of having .css